CHANGELOG.md

```diff
@@ -1,5 +1,246 @@
 # Changelog
 
+## v0.63.0
+
+### Breaking
+
+- Multiple deprecations and breaking changes to TextInputState. Use native component refs instead of react tags ([6286270e4c](https://github.com/facebook/react-native/commit/6286270e4cb10b40cfd7c8193e31d965f6815150) by [@TheSavior](https://github.com/TheSavior))
+- The `target` field of events is now a native component, not a react tag ([3b813cade1](https://github.com/facebook/react-native/commit/3b813cade1f5d6f248a39f6bbd983f68c5794fe6) by [@TheSavior](https://github.com/TheSavior))
+- Modal: Remove support for `animated` prop (deprecated in 0.26) ([1e9db7bd6d](https://github.com/facebook/react-native/commit/1e9db7bd6df3055b9b81d23f15a54bb250621a41) by [@TheSavior](https://github.com/TheSavior))
+- Alert: Remove deprecated support for passing callback as fourth argument to Alert.prompt (deprecated in 0.59) ([a26d622d04](https://github.com/facebook/react-native/commit/a26d622d04451d6872eed2491e5d3f7d4689824d) by [@TheSavior](https://github.com/TheSavior))
+- Switch: Remove support for thumbTintColor, tintColor, onTintColor props (deprecated in 0.57) ([26912bd979](https://github.com/facebook/react-native/commit/26912bd9798aeb38931466b8ddcd3a48973b0528) by [@TheSavior](https://github.com/TheSavior))
+- Bump supported Node engines to >= 10 ([f0c7178a3a](https://github.com/facebook/react-native/commit/f0c7178a3a24e7694b765946f0d884104c8cfa4c) by [@safaiyeh](https://github.com/safaiyeh))
+
+### Added
+
+- Implement `nativePerformanceNow` and `performance.now()` ([232517a574](https://github.com/facebook/react-native/commit/232517a5740f5b82cfe8779b3832e9a7a47a8d3d) by [@emilisb](https://github.com/emilisb))
+- Deleting marker cancel 2/N. [DO NOT LAND] ([c38318d084](https://github.com/facebook/react-native/commit/c38318d08469ad30535d970328704dc9ae6bb785) by [@mkareta](https://github.com/mkareta))
```

Comment:
This has an internal changelog field internally but it got removed when exported.